### ISSUE TITLE
LaunchUri not working since 1.8.0

### ISSUE THREAD

**Issue by trusis666** [fusetools/fuselibs-public#1134]
Reported in [forum](https://www.fusetools.com/community/forums/bug_reports/launchuri_not_working_since_18).

Tested on iPhone 7 / 11.2 and Nexus 5 / 6.0

## Issue

Since version 1.8.0 `tel` is not working on both mobile platforms.
Error from Xcode testing with iPhone 7:
```
Error: Uno.NullReferenceException: Object reference was null
   at Uno.Runtime.Implementation.Internal.FormatStringItem.ToString(object[])
   at string.Format(string,object[])
   at Uno.Net.Http.Uri.get_AbsoluteUri()
   at Fuse.LauncherImpl.InterAppLauncher.LaunchUri(Uno.Net.Http.Uri)
   at Fuse.Triggers.Actions.LaunchUri.Perform(Fuse.Node)
   at Fuse.Triggers.Actions.TriggerAction.PerformFromNode(Fuse.Node)
   at Fuse.Triggers.Trigger.DeferredItem.Perform()
   at Fuse.UpdateAction.Invoke()
   at Fuse.UpdateManager.ProcessDeferredActions(Fuse.Stage,Uno.Collections.List<Uno.Exception>&)
   at Fuse.UpdateManager.Update(Fuse.Stage)
   at Fuse.UpdateManager.ProcessStages()
   at Fuse.UpdateManager.Update()
   at Fuse.App.OnUpdate()
   at Outracks.Simulator.Client.Application.OnUpdate()
   at Fuse.App.OnTick(object,Uno.Platform.TimerEventArgs)
   at Uno.Platform.Display.OnTick(Uno.Platform.TimerEventArgs)
   at Uno.Platform.Displays.TickAll(Uno.Platform.TimerEventArgs)
```

In version 1.7.3 `tel` works as expected on both mobile platforms.

## Code to reproduce:

```
<App>
<StackPanel Margin="20">
    <Button Margin="10" Text="Click to call">
        <Clicked>
            <LaunchUri Uri="tel:12345678" />
        </Clicked>
    </Button>
</StackPanel>
</App>
```

Looks like regression.

**Comment by raphagodoi** [comment]
This problem happen with whatsapp url too:

whatsapp://send?text=theText;

Also tested on iOS 11, and Android 7.1

**Comment by raphagodoi** [comment]
I've rolled back to 1.7.3, did the uno clean... but the error still there, only on Fuse 1.7.0 everything works right.

```shell
3-29 09:23:14.592  9908 10505 W System.err: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat=whatsapp://phone=+55(61) 98181-9533&send?text=Este é um convite pra abrir meu portão:  https://iportao.com/app/share/or3exg78bx7pk49l }
03-29 09:23:14.592  9908 10505 W System.err: 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1899)
03-29 09:23:14.592  9908 10505 W System.err: 	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1589)
03-29 09:23:14.592  9908 10505 W System.err: 	at android.app.Activity.startActivityForResult(Activity.java:4228)
03-29 09:23:14.592  9908 10505 W System.err: 	at android.app.Activity.startActivityForResult(Activity.java:4187)
03-29 09:23:14.592  9908 10505 W System.err: 	at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:842)
03-29 09:23:14.592  9908 10505 W System.err: 	at android.app.Activity.startActivity(Activity.java:4515)
03-29 09:23:14.592  9908 10505 W System.err: 	at android.app.Activity.startActivity(Activity.java:4483)
03-29 09:23:14.592  9908 10505 W System.err: 	at com.foreign.Fuse.Android.Bindings.AndroidDeviceInterop.LaunchIntent324(AndroidDeviceInterop.java:42)
03-29 09:23:14.592  9908 10505 W System.err: 	at com.Bindings.ExternedBlockHost.LaunchIntent324(ExternedBlockHost.java:1508)
03-29 09:23:14.593  9908 10505 F libc    : Fatal signal 11 (SIGSEGV), code 1, fault addr 0x7 in tid 10505 (Thread-2)
03-29 09:23:14.594   466   466 W         : debuggerd: handling request: pid=9908 uid=10141 gid=10141 tid=10505
03-29 09:23:14.709 10783 10783 F DEBUG   : pid: 9908, tid: 10505, name: Thread-2  >>> com.iportao.android.beta <<<
03-29 09:23:15.840   466   466 W         : debuggerd: resuming target 9908
03-29 09:23:15.887  1541 13929 I ActivityManager: Process com.iportao.android.beta (pid 9908) has died
03-29 09:23:15.887  1541 13929 D ActivityManager: cleanUpApplicationRecord -- 9908
03-29 09:23:15.935   779   779 I Zygote  : Process 9908 exited due to signal (11)
`

**Comment by kusma** [comment]
@mortoray: Hmm, yeah. This is my fault. I'll look into cooking up a patch.

**Comment by kusma** [comment]
This was fixed in Fuse 1.8.2.
